bot/src/bot.rs

```diff
@@ -141,20 +142,16 @@ impl Bot {
             None,
             None,
         )?;
+
         let mut fetch_instances_count = 0;
 
         let ref_contract = self.apr_reference_contract.clone();
 
         let ver_req = VersionReq::parse(VERSION_REQ).unwrap();
+        log!(Level::Debug, "version requirement: {ver_req}");
         for app_info in saving_modules.modules {
             let version = app_info.module.info.version.to_string();
-            // Skip if version mismatches
-            if semver::Version::parse(&version)
-                .map(|v| !ver_req.matches(&v))
-                .unwrap_or(false)
-            {
-                continue;
-            }
+
```

Comment:
Why would we want to fetch them even if we can't autocompound it?

Reply:
Because then we can see how many contracts are not up to date
When there is a vulnerability in a version, we can see how much funds are still in the vulnerable code.
Overall it's a good way to be able to view our fleet.